.github/workflows/run.yml

```diff
@@ -0,0 +1,53 @@
+name: Run server
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  # Label of the container job
+  container-job:
+    # Containers must run in Linux based operating systems
+    runs-on: ubuntu-latest
+    # Docker Hub image that `container-job` executes in
+    container: node:10.18-jessie
+
+    # Service containers to run with `container-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test_observation
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          npm install
+          cd action
+          npm install
+
+      - name: Start server
+        run: |
+          npm run start &
+        env:
+          DATABASE_URL: postgresql:///test_observation
```

Comment:
This doesn't look like a valid postgres URL, I would have expected two slashes in the schema and a username?

Reply:
Maybe. I use `file:///foo` a lot so maybe I just default to it.